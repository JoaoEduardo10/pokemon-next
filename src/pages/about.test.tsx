import { render } from '@testing-library/react';
import About from './about';

describe('<About />', () => {
	it('should', () => {
		const { container } = render(<About />);

		expect(container).toMatchSnapshot();
	});
});