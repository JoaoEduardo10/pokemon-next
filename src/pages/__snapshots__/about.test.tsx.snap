// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should 1`] = `
<div>
  <div
    class="about"
  >
    <h1>
      Sobre o projeto
    </h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Minima ipsum quidem esse, ab ipsa repellat facilis doloribus ea ad consequuntur corrupti, autem, enim maxime tempora amet? Accusantium officia consequatur deleniti.
    </p>
    <img
      alt="charizard"
      data-nimg="1"
      decoding="async"
      height="300"
      loading="lazy"
      src="/_next/image?url=%2Fimages%2Fcharizard.png&w=640&q=75"
      srcset="/_next/image?url=%2Fimages%2Fcharizard.png&w=384&q=75 1x, /_next/image?url=%2Fimages%2Fcharizard.png&w=640&q=75 2x"
      style="color: transparent;"
      width="300"
    />
  </div>
</div>
`;
